app/models/volunteer.rb

```diff
@@ -71,6 +73,12 @@ class Volunteer < ApplicationRecord
     joins(:assignments).merge(Assignment.active_between(start_date, end_date)).distinct
   }
   scope :without_assignment, (-> { left_outer_joins(:assignments).where(assignments: { id: nil }) })
+  scope :without_active_assignment, (-> { joins(:assignments).merge(Assignment.ended) })
+  scope :not_responsible, (-> { where.not(id: GroupOffer.pluck(:responsible_id)) })
+  scope :not_in_any_group_offer, lambda {
+    joins('LEFT JOIN group_offers_volunteers ON volunteers.id = group_offers_volunteers.volunteer_id')
+      .where('group_offers_volunteers.volunteer_id IS NULL')
```

Comment:
`.where(group_offers: { volunteer_id: nil })` should do the same

Reply:
That looks for a `volunteer_id` at the `group_offers`, but there isn't any since the relationship is many to many. That's why I use the join table.

Reply:
Ah sorry I meant `.where(group_offers_volunteers: { volunteer_id: nil })` (I thought you can specify the association name but it's indeed the table name)